src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from "react-router-dom"
+import NavLogo from "../../Assets/nav-logo.png"
+
+const Navbar = () => {
+    return (
+        <nav class="navbar">
+            <div class="nav-logo">
+                <img src={NavLogo} alt="logo" />
+            </div>
+            <div class="nav-items">
+                <Link to="/" class="nav-link sm-title hide-md" href="./">About Us</Link>
+                <a class="nav-link sm-title hide-md" href="./Products/products.html">Shop</a>
+            </div>
```

Comment:
changes: you can replace < a > tags with < Link >.

Reply:
will do it in the shoppage branch